src/lib/selectCylinderSets.ts

```diff
@@ -0,0 +1,37 @@
+import { Knex } from 'knex';
+import { Cylinder, CylinderSet } from '../types/cylinderSet.types';
+import { log } from './log';
+
+export default async function selectCylinderSets(
+  trx: Knex.Transaction
+): Promise<CylinderSet[] | undefined> {
+  const set: CylinderSet[] | undefined = await trx(
+    'diving_cylinder_set'
+  ).select<CylinderSet[]>('id', 'owner', 'name');
+
+  for (const singleSet of set) {
+    singleSet.cylinders = await trx('diving_cylinder')
+      .innerJoin(
+        'diving_cylinder_to_set',
+        'diving_cylinder.id',
+        '=',
+        'diving_cylinder_to_set.cylinder'
+      )
+      .select<Cylinder[]>(
+        'id',
+        'volume',
+        'pressure',
+        'material',
+        'serial_number as serialNumber',
+        'inspection'
+      )
+      .where('diving_cylinder_to_set.cylinder_set', singleSet.id);
```

Comment:
Tää on nyt ihan sama koodi kuin täällä:
https://github.com/Tampereen-Urheilusukeltajat/Blenderi-Backend/blob/9e7297fd1dd2d23303e6bb78d198313cda6e9373/src/lib/selectCylinderSet.ts#L19-L34

Näitä kahta apufunktiota pitäisi refactoroida niin, ettei toisteta itseämme.